python/lsst/afw/image/apCorrMap.cc

```diff
@@ -32,14 +40,62 @@ PYBIND11_PLUGIN(_apCorrMap) {
     py::module mod("_apCorrMap", "Python wrapper for afw _apCorrMap library");
 
     /* Module level */
+    lsst::afw::table::io::declarePersistableFacade<ApCorrMap>(mod, "ApCorrMap");
+    py::class_<ApCorrMap,
+               std::shared_ptr<ApCorrMap>,
+               lsst::afw::table::io::PersistableFacade<ApCorrMap>,
+               lsst::afw::table::io::Persistable>
+        clsApCorrMap(mod, "ApCorrMap");
 
     /* Member types and enums */
 
     /* Constructors */
+    clsApCorrMap.def(py::init<>());
 
     /* Operators */
+    // TODO: pybind11 ApCorrMap's C++ operators should return ApCorrMap &
+    clsApCorrMap.def("__imul__", [](ApCorrMap & self, double const scale) {
+        self *= scale;
+        return self;
+    });
+    clsApCorrMap.def("__idiv__", [](ApCorrMap & self, double const scale) {
+        self /= scale;
+        return self;
+    });
+    clsApCorrMap.def("__itruediv__", [](ApCorrMap & self, double const scale) {
+        self /= scale;
+        return self;
+    });
 
     /* Members */
+    clsApCorrMap.def("get", &ApCorrMap::get);
+    clsApCorrMap.def("set", &ApCorrMap::set);
+
+    // I couldn't figure out how to get Swig to expose the C++ iterators using std_map.i and (e.g.)
+    // code in utils.i; it kept wrapping the iterators as opaque objects I couldn't deference, probably
+    // due to some weirdness with the typedefs.  I don't want to sink time into debugging that.
+    // So I just wrote this function to return a list of names, and I'll base the Python iterators on
+    // that. -- Russell Owen
+    // TODO: pybind11 replace this with pybind11's support for iterators/slicing/etc.
+    clsApCorrMap.def("keys", [](ApCorrMap const & self) {
+        // Can't create a vector of const element type, not sure why
+        using KeyType = std::remove_const<ApCorrMap::Iterator::value_type::first_type>::type;
+        auto r = std::vector<KeyType>();
+        r.reserve(self.size());
+        for (ApCorrMap::Iterator i = self.begin(); i != self.end(); ++i) {
+            r.push_back(i->first);
+        }
+        return r;
+    });
```

Comment:
`keys` itself is not an iterator, it creates a copy of the map's keyset as a list. This means iterators are unaffected by the deallocation of the parent object, at the cost of silently failing any attempt to modify the keys.

This is by no means a good long-term solution, but it's a straight copy of what the swig interface did, so it works no worse than the old code.